lib/trento/users.ex

```diff
@@ -34,6 +34,8 @@ defmodule Trento.Users do
     |> Repo.all()
   end
 
+  def admin_username, do: Application.fetch_env!(:trento, :admin_username)
```

Comment:
question: Must this be public?
I mean, if it is only for tests, you know that the value is `admin` coming from the config file

Reply:
done!